tests/testsuite/cargo_tree/deps.rs

```diff
@@ -57,7 +57,8 @@ foo v0.1.0 ([ROOT]/foo)
     └── b v1.0.0 (*)
 [dev-dependencies]
 └── devdep v1.0.0
-    └── b v1.0.0 (*)
+    └── b v1.0.0
+        └── c v1.0.0
```

Comment:
This seems unrelated, why did it change?

Reply:
I think you put the finger on it and it's probably caused by the Eq/Ord mismatch on NodePackage, testing it out now :)